CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # Changelog
 All notable changes to this project will be documented in this file.
+
+## [5.13](https://github.com/Backbase/stream-services/compare/5.12.0...5.13.0)
```

Comment:
`5.13.0` ?

Reply:
I updated to `5.14.0` now, I did not rebase with the `05.13.0` that were committed that's why I ended up setting to it.